src/plugins/unified_search/public/filters_builder/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { FiltersBuilder } from './filters_builder';
+
+export type { FiltersBuilderProps } from './filters_builder';
+// React.lazy support
+// eslint-disable-next-line import/no-default-export
+export default FiltersBuilder;
```

Comment:
why did you remove `React.lazy` wrapper?  please refactor this file in a manner with `src/plugins/unified_search/public/filter_bar/index.tsx `